content/doc/tutorials/_download-and-run-jenkins-in-docker.adoc

```diff
@@ -30,7 +30,7 @@ later) by running the command: +
   through the tutorial) without you losing any work you've already done.
 . Start the downloaded Jenkins/Blue Ocean Docker image (for running this
 tutorial) as a Docker container using the appropriate command:
-** *On Mac OS X and Linux:*
+** *On macOS and Linux:*
```

Comment:
🤦‍♂️ ... Looks like I'm 4 years out of date (https://en.wikipedia.org/wiki/MacOS).